src/CrashlogFile.spec.ts

```diff
@@ -4,6 +4,9 @@ import { util as bscUtil, standardizePath as s } from 'brighterscript';
 import { CrashlogFile } from './CrashlogFile';
 import { Runner } from './Runner';
 import type { RunnerOptions } from './interfaces';
+// TODO: How to remove this? No quickfix.
```

Comment:
You just have to move the line up and down until it's sorted alphabetically and the error goes away.